ccd-definition/CaseTypeTab.json

```diff
@@ -47,7 +37,7 @@
     "TabLabel": "People in the case",
     "TabDisplayOrder": 2,
```

Comment:
Defo 2 ?

Reply:
no this should be 4. Weird that's got so out of sync 🤔